corehq/apps/cloudcare/static/cloudcare/js/backbone/apps.js

```diff
@@ -134,7 +134,9 @@ hqDefine('cloudcare/js/backbone/apps.js', function () {
             return this.get("urlRoot");
         },
         getSubmitUrl: function () {
-            return this.get('post_url');
+            // http://stackoverflow.com/a/2599721/835696
```

Comment:
this wasn't a super highly voted answer, but i think it works fine for us. essentially captures anything that isn't a `/` and deletes it